pkg/infra/iac2/templates/origin_access_identity/factory.ts

```diff
@@ -0,0 +1,13 @@
+import * as aws from '@pulumi/aws'
+
+interface Args {
+    Name: string
+    Comment: string
+}
+
+// noinspection JSUnusedLocalSymbols
+function create(args: Args): aws.cloudfront.OriginAccessIdentity {
+    return new aws.cloudfront.OriginAccessIdentity(args.Name, {
+        comment: args.Comment,
```

Comment:
Should we add `// comment`?

(No actual need, I just wanted to add a comment to add a comment to the comment Comment.)